### PR TITLE
Add more test case scenario for @psalm-import-type

### PR DESCRIPTION
**Pull request by malukenho**

### CODE REVIEW

File: tests/TypeAnnotationTest.php [hunk 1]
```diff
@@ -189,6 +189,46 @@ function toArray(): array {
...
+                    /** @psalm-type PhoneType = array{phone: string} */
+                    class Phone {
+                        /** @psalm-return PhoneType */
+                        public function toArray(): array {
+                            return ["phone" => "Nokia"];
+                        }
+                    }
+
+                    /**
+                     * @psalm-import-type PhoneType from Phone
```
Comment (staabm) [inline tests/TypeAnnotationTest.php:223]: „Phone“ is a class-name , I guess. Is a test required with a fqcn?

Comment (malukenho) [reply tests/TypeAnnotationTest.php:223]: Added on https://github.com/vimeo/psalm/pull/3375/commits/24b2dbc716a5a693185d222e5cef078a0b48eb59

**Comment by malukenho** [comment]
@muglug it is ready to go! I've done a fix for imported types here as well.

**Comment by malukenho** [comment]
Added some failing cases

**Comment by muglug** [comment]
Thanks!

**Pull request merged**